game/MiiStudio_Miidatafile.ksy

```diff
@@ -0,0 +1,142 @@
+  - id: face_makeup
+    type: u1
+    doc: Face makeup. Ranges from 0 to 11.
+  - id: face_type
+    type: u1
+    doc: Face shape. Ranges from 0 to 11. Not ordered the same as visible in editor.
+  - id: face_wrinkles
+    type: u1
+    doc: Face wrinkles. Ranges from 0 to 11.
+  - id: favorite_color
+    type: u1
+    doc: Favorite color. Ranges from 0 to 11.
+  - id: gender
+    type: u1
+    doc: Mii gender. 0 = male, 1 = female.
+  - id: glasses_color
+    type: u1
+    doc: Glasses color. Ranges from 0 to 99. Not ordered the same as visible in editor.
+  - id: glasses_size
+    type: u1
+    doc: Glasses size. Ranges from 0 to 7, small to large.
+  - id: glasses_type
+    type: u1
+    doc: Glasses type. Ranges from 0 to 19. Not ordered the same as visible in editor.
+  - id: glasses_vertical
+    type: u1
+    doc: Glasses Y (vertical) position. Ranges from 20 to 0, low to high.
+  - id: hair_color
+    type: u1
+    doc: Hair color. Ranges from 0 to 99. Not ordered the same as visible in editor.
+  - id: hair_flip
+    type: u1
+    doc: Flip hair. 0 = no, 1 = yes.
+  - id: hair_type
+    type: u1
+    doc: Hair type. Ranges from 0 to 131. Not ordered the same as visible in editor.
+  - id: body_height
+    type: u1
+    doc: Body height. Ranges from 0 to 127, short to tall.
+  - id: mole_size
+    type: u1
+    doc: Beauty mark size. Ranges from 0 to 8, small to large.
+  - id: mole_enable
+    type: u1
+    doc: Enable beauty mark. 0 = no, 1 = yes.
+  - id: mole_horizontal
+    type: u1
+    doc: Beauty mark X (horizontal) position. Ranges from 0 to 16, left to right.
+  - id: mole_vertical
+    type: u1
+    doc: Beauty mark Y (vertical) position. Ranges from 30 to 0, low to high.
+  - id: mouth_stretch
+    type: u1
+    doc: Mouth stretch. Ranges from 0 to 6, small to large.
+  - id: mouth_color
+    type: u1
+    doc: Mouth color. The default colors are ordered the same as visible in editor, ranging from 19 to 23. The custom colors are not and range from 0 to 99.
+  - id: mouth_size
+    type: u1
+    doc: Mouth size. Ranges from 0 to 8, small to large.
+  - id: mouth_type
+    type: u1
+    doc: Mouth type. Ranges from 0 to 35. Not ordered the same as visible in editor.
+  - id: mouth_vertical
+    type: u1
+    doc: Mouth Y (vertical) position. Ranges from 18 to 0, low to high.
+  - id: beard_size
+    type: u1
+    doc: Mustache size. Ranges from 0 to 8, small to large.
+  - id: beard_mustache
+    type: u1
+    doc: Mustache type. Ranges from 0 to 5.
+  - id: beard_vertical
+    type: u1
+    doc: Mustache Y (vertical) position. Ranges from 16 to 0, low to high.
+  - id: nose_size
+    type: u1
+    doc: Nose size. Ranges from 0 to 8, small to large.
+  - id: nose_type
+    type: u1
+    doc: Nose type. Ranges from 0 to 17. Not ordered the same as visible in editor.
+  - id: nose_vertical
+    type: u1
+    doc: Nose Y (vertical) position. Ranges from 18 to 0, low to high.
```

Comment:
Minor thing: there should be a newline at the end of the file.

(Same for the other two files.)